sonic-ycabled/tests/t0-sample-port-config.ini

```diff
@@ -0,0 +1,14 @@
+# name          lanes             alias
```

Comment:
where is this file being used in y-cable tests?

Reply:
fixed